app/controllers/projects_controller.rb

```diff
@@ -1,5 +1,21 @@
 class ProjectsController < ApplicationController
-  def review_turnaround
-    @metrics = Queries::DailyMetrics.call(params[:project_id])
+  before_action :set_params
+
+  def user_project_metric
+    if valid_params?
+      @metrics = Queries::DailyMetrics.call(params[:project_id], params[:metric_type])
+    else
+      render file: 'public/404.html', status: :not_found
+    end
+  end
+
+  private
+
+  def set_params
```

Comment:
I don't think this is doing anything 🤔 Strong params are used for creating/updating objects mostly right?